### PR TITLE
fix: tiptap issues

### PR DESCRIPTION
**Pull request by sandren**
This PR adds a trailing node to the editor when the last node isn't a paragraph, making it possible to still add content where the existing gap cursor cannot reach.

### CODE REVIEW

File: apps/web/partials/editor/editor.tsx [hunk 1]
```diff
@@ -163,16 +164,9 @@ export const Editor = React.memo(function Editor({
...
-
   return (
-    <div>
+    <div className={cx(editable ? 'editable' : 'not-editable')}>
       <EditorContent editor={editor} />
-      <FloatingMenu editor={editor}>
-        <div className="absolute -left-12 -top-3">
-          <SquareButton onClick={openCommandMenu} icon={<Plus />} />
-        </div>
-      </FloatingMenu>
```
Comment (baiirun) [inline apps/web/partials/editor/editor.tsx:175]: Did you mean to delete the FloatingMenu?

File: apps/web/partials/editor/editor.tsx [hunk 2]
```diff
@@ -163,16 +164,9 @@ export const Editor = React.memo(function Editor({
 
   if (!editor) return null;
 
-  const openCommandMenu = () => editor?.chain().focus().insertContent('/').run();
-
   return (
-    <div>
+    <div className={cx(editable ? 'editable' : 'not-editable')}>
```
Comment (baiirun) [inline apps/web/partials/editor/editor.tsx:168]: What does this do?

File: apps/web/partials/editor/editor.tsx (hunk 1 above)
Comment (sandren) [reply apps/web/partials/editor/editor.tsx:175]: Yes, that's intentional per Nate.

File: apps/web/partials/editor/editor.tsx (hunk 2 above)
Comment (sandren) [reply apps/web/partials/editor/editor.tsx:168]: It lets us [hide the new trailing node](https://github.com/geobrowser/geogenesis/pull/685/files#diff-fb8f7a66556e842843c02062f263ee7885171f196c2d578c1d53a11a1a2a4eacR68-R71) when in browse mode in a way that doesn't cause layout shift.

File: apps/web/partials/editor/editor.tsx (hunk 1 above)
Comment (baiirun) [reply apps/web/partials/editor/editor.tsx:175]: Do you have a link to the designs? Just so I have all the context

**Review by baiirun — APPROVED** [review]

**Pull request merged**